### DISCUSSION TITLE
index

### DISCUSSION THREAD

**Discussion by giscus** [Frankoxer/frankoxer.github.io#3] [Announcements]
# index

Fanovian 的笔记本

https://note.fanovian.cc

**Comment by Flyburg** [comment]
自己先占个坑😘

**Comment by EternalLightning** [comment]
巨！

**Comment by jerryfeng2003** [comment]
巨！

**Reply by Flyburg** [reply]
大神啊

**Comment by Argonding** [comment]
巨！

**Comment by Wcowin** [comment]
简约大气，来加个友链~

**Reply by Flyburg** [reply]
哈喽～上次在贵站加上了，今天我这里ui大更新刚做上友链界面，有兴趣可以来这里加哦～

友链页面
https://note.fanovian.cc/link/friends/

友链添加指南
https://note.fanovian.cc/link/join/
